execution/block-partitioner/src/v2/config.rs

```diff
@@ -51,7 +51,7 @@ impl PartitionerV2Config {
 impl Default for PartitionerV2Config {
     fn default() -> Self {
         Self {
-            num_threads: 8,
+            num_threads: 4,
```

Comment:
I actually never tried running V2 with default config. I always use 32 threads 🤣

Reply:
The executor benchmark is using the default config right?

Reply:
maybe. Anyway I always overwrite it... All my numbers are from runs with 32 threads.

Reply:
I tried it out and the partitioning performance is fine with 4 threads, this is going to reduce some thread contention so less thread is better.

Reply:
ok i will also re-run with 4 threads and see if there's any TPS diff.